src/main/java/org/example/bookstore/dto/orderdto/OrderUpdatedDto.java

```diff
@@ -0,0 +1,5 @@
+package org.example.bookstore.dto.orderdto;
+
+public class OrderUpdatedDto {
+    private String status;
```

Comment:
should work and validate without a problem, maybe add @JsonCreator with a `Status fromString(String str)` method to the enum class if it does not
```suggestion
    private Status status;
```

Reply:
also add validation
```suggestion
    @NotNull
    private String status;
```